hasura/metadata/databases/staging/tables/public_pending_token_gifts.yaml

```diff
@@ -0,0 +1,77 @@
+table:
+  name: pending_token_gifts
+  schema: public
+object_relationships:
+- name: circle
+  using:
+    foreign_key_constraint_on: circle_id
+- name: epoch
+  using:
+    manual_configuration:
+      column_mapping:
+        epoch_id: id
+      insertion_order: null
+      remote_table:
+        name: epoches
+        schema: public
+- name: private_received
+  using:
+    manual_configuration:
+      column_mapping:
+        recipient_id: recipient_id
+      insertion_order: null
+      remote_table:
+        name: pending_gift_private
+        schema: public
+- name: private_sent
+  using:
+    manual_configuration:
+      column_mapping:
+        sender_id: sender_id
+      insertion_order: null
+      remote_table:
+        name: pending_gift_private
+        schema: public
+- name: recipient
+  using:
+    foreign_key_constraint_on: recipient_id
+- name: sender
+  using:
+    foreign_key_constraint_on: sender_id
+select_permissions:
+- permission:
+    columns:
+    - id
+    - sender_id
+    - sender_address
+    - recipient_id
+    - recipient_address
+    - tokens
+    - note
+    - dts_created
+    - created_at
+    - updated_at
+    - circle_id
+    - epoch_id
+    filter: {}
+  role: superadmin
+- permission:
+    columns:
+    - circle_id
+    - created_at
+    - dts_created
+    - epoch_id
+    - id
+    - recipient_address
+    - recipient_id
+    - sender_address
+    - sender_id
+    - tokens
+    - updated_at
+    filter:
+      circle:
+        users:
+          profile:
+            id:
+              _eq: X-Hasura-User-Id
```

Comment:
Are pending gifts supposed to be visible by all members of the circle? If not then this might need to be changed, but if so its fine.

Reply:
Yes they can see so the map of the graph can be constructed. Pending gifts are all the gifts sent in the current active epoch. Only the notes are restricted hence the private view